firestore/google/cloud/firestore_v1beta1/_helpers.py

```diff
@@ -131,6 +126,13 @@ def __init__(self, *parts):
                 raise ValueError(error)
         self.parts = tuple(parts)
 
+    def __repr__(self):
+        paths = ""
+        for part in self.parts:
+            paths += "'" + part + "',"
```

Comment:
I would encourage you to just call `to_api_repr`. Field Paths in the API representation are in the simple case more readable (a.b versus 'a','b') and in the complex case more correct('a'.b versus ''a''.b).

Reply:
I thought about doing something similar at first, but ideally __repr__ in python returns the construction of the object in string form.  In other words, instead of duplicating the functionality of `to_api_repr()`,
`repr(FieldPath("a", "b"))` should return a string `FieldPath("a", "b")`

Reply:
Sounds good.